dags/walg_backup_dags.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+# DAG to make a full backup of a postgres database to a gcs bucket.
+"""
+from walg_backups import create_backup_task
+from dag_configuration import default_dag_args
+from datetime import datetime, timedelta
+from airflow import DAG
+import os
+import sys
+sys.path.insert(0, os.path.abspath(os.path.dirname(__file__)))
+
+
+START_DATE = datetime.now() - timedelta(days=2)
+
+ciip_namespace = os.getenv('CIIP_NAMESPACE')
+ggircs_namespace = os.getenv('GGIRCS_NAMESPACE')
+in_cluster = os.getenv('LOCAL_AIRFLOW', False) == 'False'
+
+default_args = {
+    **default_dag_args,
+    'start_date': START_DATE
+}
+
+DAG_ID = os.path.basename(__file__).replace(".pyc", "").replace(".py", "")
```

Comment:
Maybe we don't need to do this filename dag naming anymore? The name dags doesn't really need to be in the dag id.

Reply:
But then if we'll soon remove this file and put these dags in different repos I guess it's fine.. 🤔

Reply:
Ah right, I renamed the file to avoid confustion. We can hard code it for now